### ISSUE TITLE
Update rustls dependency target

### ISSUE THREAD

**Issue by detly** [ctz/rustls-native-certs#13]
I just tried to use this crate with [rumqttc](https://github.com/bytebeamio/rumqtt/tree/master/rumqttc), and got:

```text
error[E0308]: try expression alternatives have incompatible types
 --> src/main.rs:9:28
  |
9 |     tlsconfig.root_store = rustls_native_certs::load_native_certs()?;
  |                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected struct `rustls::anchors::RootCertStore`, found a different struct `rustls::anchors::RootCertStore`
  |
  = note: perhaps two different versions of crate `rustls` are being used?
```

Indeed this crate depends on rustls 0.19, and rumqttc depends on rustls 0.2. Would you be open to targeting the same?

**Comment by detly** [comment]
Arg, it's using tokio-rustls, not vanilla rustls. I'm not really sure what to do about that then.

**Comment by detly** [comment]
It actually depends on tokio-rustls, not rustls, but that resolves to rustls 0.7.0. I'm just sorting out some other dependency issues and then see if I can provide a PR.

**Comment by detly** [comment]
Okay, sorry for all the comment/delete noise. I'm new to Cargo dependency wrangling. I finally realise that I've got it backwards: your crate depends on rustls 0.19 and rumqttc depends on **tokio-rustls 0.2**, which indirectly depends on an **older** version of rustls. I will open a PR there instead of here.
